### PR TITLE
Fix /logout docs

### PR DESCRIPTION
**Pull request by benClmnt**

### CODE REVIEW

File: docs/UsersAPI.md [hunk 1]
```diff
@@ -67,12 +67,15 @@ Body:
 ### POST `/logout`
 
 Logout.
+> :warning: **Requires authorization**
 
 #### Request:
 
 Body: 
 ```
-{}
```
Comment (stevenwjy) [inline docs/UsersAPI.md]: The token should be put in the header

**Comment by donjar** [comment]
Use Swagger? :smile:

https://swagger.io/

File: docs/UsersAPI.md (hunk 1 above)
Comment (benClmnt) [reply docs/UsersAPI.md]: Nope, it should be put in both header and body

Comment (benClmnt) [reply docs/UsersAPI.md]: at least for the code at this point of talking

Comment (stevenwjy) [reply docs/UsersAPI.md]: Ups, sorry, just realized this...
Okay, will use this request format for the time being

**Pull request merged**